replay: ref onboarding snippet (#5926)
getsentry/sentry-docs: src/platform-includes/session-replay/install-session-replay/javascript.mdx

> Install the Replay package with NPM or your favorite package manager. Alternatively, you can load the Replay integration via a CDN bundle.

Reviewer: TBH I'm not even sure I'd use the word "favorite" here. Maybe something more like "package manager of your choice."

> Install the Replay package with NPM or your favorite package manager. Alternatively, you can load the Replay integration via a CDN bundle.

Author: ```suggestion
Install the Replay package with the package manager of your choice. Alternatively, you can load the Replay integration via a CDN bundle.
```

> Install the Replay package with NPM or your favorite package manager. Alternatively, you can load the Replay integration via a CDN bundle.

Author: Thanks, will do
